### DISCUSSION TITLE
禾煜科技助涮乃葉導入送餐機器人，大幅翻轉職場環境與顧客服務體驗

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#599] [General]
https://www.ithome.com.tw/pr/149980
下面開放約好友吃涮乃葉

**Comment by Raywei0214** [comment]
卡耐基說過一句發人省思的話，我們若已接受最壞的，就再沒有什麼損失。這讓我的思緒清晰了。如果此時我們選擇忽略火鍋，那後果可想而知。

**Comment by awehao** [comment]
我還是喜歡看服務生那親切的笑臉

**Comment by larry40305** [comment]
以後就會朝人形送餐機器人發展了

**Comment by manipulateed** [comment]
其實現在很多餐廳都看的見這樣的機器人出現，而且他們還能準確地避開客人，又能達到為服務生提供更多充裕的時間以此來緩解人手不足的問題，這個新潮的科技也算是餐廳的一個賣點，期待有更多的運用像是帶位。

**Comment by Valentinaa0214** [comment]
海底撈也有這種送餐機器人！但大部分的餐廳還是會由店員將餐點從機器人手上送到顧客桌上，在環境比較狹小的門市機器人也很容易擋到店員或客人的動線，實際問題似乎還是有點多...

**Comment by yjpxi** [comment]
現在很多店家都有送餐機器人，送餐機器人為餐飲業提供了提升服務效率和顧客服務體驗的機會，並且能改善職場環境；然而，這也伴隨著高成本投入、技術挑戰和人力資源影響等挑戰。
